webapp/src/components/ProgramCalendar.tsx

```diff
@@ -69,6 +69,26 @@ const ProgramCalendar = ({ program }: ProgramCalendarProps) => {
         minHeight: '4%',
       };
     }
+    if (event.allDay && currentView !== 'agenda') {
+      style = {
+        backgroundColor: 'midnightblue',
+      };
+    }
+
+    if (event.allDay && currentView === 'week') {
+      style = {
+        height: '190%',
+        backgroundColor: 'midnightblue',
+      };
+    }
+
+    if (event.allDay && currentView === 'day') {
+      style = {
+        height: '220%',
+        backgroundColor: 'midnightblue',
+      };
+    }
+
```

Comment:
Personally I won't change the height of all day events since if we have multiple all day events in one day it's gonna take up a lot of space